### ISSUE TITLE
buildOpamPackage: missing LIBDIR

### ISSUE THREAD

**Issue by kamilchm** [timbertson/opam2nix-packages#47]
I'm trying to build [esy](https://github.com/esy/esy). It needs custom source for some packages eg. [cmdliner](https://github.com/esy/esy/blob/4d840fe96ab80aba74fb08d483757a73cd1a3d0f/esy.lock/index.json#L6220) so I wanted to use `buildOpamPackage` to [build it](https://github.com/kamilchm/nix-esy/blob/5cc6dcdd67720333fca2ba72e119cd6419d177d1/default.nix#L29-L37) but it fails with the error:
```
+ make install LIBDIR= DOCDIR=
install -d
install: missing file operand
```
so [it looks like](https://github.com/esy-ocaml/cmdliner/blob/8500634a96019c4d29b1751628025b693f2b97d6/Makefile#L53) there's no `LIBDIR` provided when executing the `installPhase`.
Any ideas what can be the problem?

**Comment by timbertson** [comment]
Yeah, I've seen this in a couple packages too. I think I know what the
issue is, just need to find some time to test the fix

**Comment by kamilchm** [comment]
I could try to fix it myself if you point me somewhere.

**Comment by timbertson** [comment]
I've pushed a commit to the `test` branch, if you're able to try that out and report back that'd be great :+1: 
https://github.com/timbertson/opam2nix/tree/test

**Comment by timbertson** [comment]
Actually, I managed to test this and it _seems_ to be working (published v0.4.17). Let me know if you still see issues

**Comment by kamilchm** [comment]
I works now. Thx!
